index.js

```diff
@@ -4,61 +4,78 @@ const childProcess = require('child_process');
 const pify = require('pify');
 
 const bin = path.join(__dirname, 'main');
-const xprop = 'xprop -id $(xprop -root 32x \'\\t$0\' _NET_ACTIVE_WINDOW | cut -f 2)';
+const xpropActive = 'xprop -root 32x \'\\t$0\' _NET_ACTIVE_WINDOW | cut -f 2';
```

Comment:
Since we're already doing most things in JS now, can you drop the `cut -f 2` part and do it in JS too?